crates/fs/private/forest.rs

```diff
@@ -3,17 +3,34 @@ use std::rc::Rc;
 use anyhow::Result;
 use libipld::Cid;
 use log::debug;
+use rand_core::RngCore;
 
 use crate::{BlockStore, HashOutput};
 
-use super::{hamt::Hamt, namefilter::Namefilter, Key, PrivateNode, PrivateRef, Rng};
+use super::{hamt::Hamt, namefilter::Namefilter, Key, PrivateNode, PrivateRef};
 
 //--------------------------------------------------------------------------------------------------
 // Type Definitions
 //--------------------------------------------------------------------------------------------------
 
-// TODO(appcypher): Change Cid to PrivateLink<PrivateNode>.
-// TODO(appcypher): And eventually to BTreeSet<PrivateLink<PrivateNode>>.
+/// PrivateForest is a HAMT that stores CIDs of private nodes using their saturated namefilters as keys.
```

Comment:
```suggestion
/// PrivateForest is a HAMT that stores CIDs of encrypted private nodes keyed by saturated namefilters.
```

I want to add the encryption thing in there since I think that may help in the overview. And needed to change the wording around "key" to make it obvious it's not encrypted using saturated namefilters, but keyed by them.